src/app/fdctl/run/tiles/fd_verify.c

```diff
@@ -135,22 +137,12 @@ after_frag( void *             _ctx,
                   payload_sz, txn_t_sz ));
   }
 
-  /* We need to access signatures and accounts, which are all before the recent_blockhash_off.
-     We assert that the payload_sz includes all signatures and account pubkeys we need. */
-  ushort recent_blockhash_off = txn_t->recent_blockhash_off;
-  if( FD_UNLIKELY( recent_blockhash_off>=*opt_sz ) ) {
-    FD_LOG_ERR( ("txn is invalid: payload_sz = %lx, recent_blockhash_off = %x", *opt_sz, recent_blockhash_off ) );
-  }
-
-  ulong txn_sig;
-  int res = fd_txn_verify( ctx, txn, (ushort)payload_sz, txn_t, &txn_sig );
+  int res = fd_txn_verify( ctx, txn, (ushort)payload_sz, txn_t );
   if( FD_UNLIKELY( res!=FD_TXN_VERIFY_SUCCESS ) ) {
     *opt_filter = 1; /* Signature verification failed. */
     return;
   }
 
-  ulong tspub = (ulong)fd_frag_meta_ts_comp( fd_tickcount() );
-  fd_mux_publish( mux, txn_sig, ctx->out_chunk, new_sz, 0UL, *opt_tsorig, tspub );
```

Comment:
Verify no longer needs to publish `txn_sig`. This value is now different for every tile (and different than inside dedup).